src/db/dal/user.ts

```diff
@@ -139,7 +149,7 @@ export const getProfileImageUploadPresignedUrl = async (keycloak_id: string) =>
     };
 };
 
-export const getUserById = async (keycloak_id: string, isOwn: boolean): Promise<IUserOuput> => {
+export const getUserById = async (keycloak_id: string, isOwn: boolean): Promise<UserModel> => {
```

Comment:
a reason to change the return type ?

Reply:
I changed it back to IUserOutput but the real return was UserModel. To keep the IUserOutput, I had to add .dataValues at the end of the return